debian-buster/miniconda3.dockerfile

```diff
@@ -0,0 +1,62 @@
+# Authors:
+#   Unai Martinez-Corral
+#   Carlos Eduardo de Paula
+#
+# Copyright 2019-2021 Unai Martinez-Corral <unai.martinezcorral@ehu.eus>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ARG REGISTRY='ghcr.io/hdl/debian-buster'
+
+#---
+
+FROM $REGISTRY/build:base AS build
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+RUN apt-get update -qq \
+    && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+        bzip2 \
+        git \
+        libglib2.0-0 \
+        libsm6 \
+        libxext6 \
+        libxrender1 \
+        mercurial \
+        subversion \
+        wget \
```

Comment:
`curl` is included in `build/base` and `build/build` already. Better use that than installing `wget`.